docs/content-api.md

```diff
@@ -0,0 +1,226 @@
+
+6. In the folder with this README.md file, run `python main.py`.
+
+7. Open a browser or use `curl` or test code to interact with
+   the server at the address and port number displayed. This is
+   usually `http://127.0.0.1:8080`.
+
+## Running Local Tests
+
+### Prepare the test identity
+
+The test suite uses *application default credentials* to get an
+identity token needed to authenticate API requests. The test suite
+authenticates every request using these credentials. We will
+create a test account and use its credentials for testing.
+
+Authenticated requests require a account that can be authenticated
+by the Google Identity Platform. Any service account can be
+used; it does not have to be a user or service account with
+permission in any specific project.
+
+You must create a test service account and download a key file
+for it. You only need to do this once to run one or more local
+tests.
+
+1. Create a test service account for a project using the
+[Google Cloud console](https://console.cloud.google.com/). Do not
+grant this service account any additional access, and (optionally)
+remove any default permissions. Note the service account's
+email address shown in the console.
+2. [Create and download a JSON service account key](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#iam-service-account-keys-create-console).
+Guard
+the downloaded key carefully. Note the full path to the key file.
+
+### Run the test suite
+
+In the directory that includes this README, open a command shell, and run
+the following commands commands below. It is recommended that you create and
+activate a [*virtualenv* virtual environment](https://virtualenv.pypa.io/en/latest/index.html)
+before doing this.
+
+    pip install -r requirements.txt
+    pip install -r requirements-test.txt
+    pytest
+
+This will run the test suite and report any failures.
+
+## Approver Seeding CLI
+
+The database will need to have an `Approver` resource with the same
+email address as the test service account. Approvers can perform all
+API operations, including designating campaign managers. This step
+will create that `Approver` resource.
+
+If you do not want to use Cloud Console to set up an approver, you can use
+the approver seeding script from any authenticated machine.
+
+1. Set the `GOOGLE_APPLICATION_CREDENTIALS` environment variable to
+   the full path you noted in the previous step.
+1. "Seed" the local Emblem API test Cloud Firestore database with this
+   service account as an approver.
+1. Navigate to `../content-api/data`
+1. Run the seeding script:
+
+        python data/seed_test_approver.py <YOUR_EMAIL_ADDRESS>
+
+You will need to do this every time you launch a new database.
+
+## Use the Cloud Firestore emulator
+
+You may use the Cloud Firestore emulator if working with data in the Cloud
+introduces too much latency. Warning: This is not a standard workflow and these
+instructions may be out of date.
```

Comment:
This feels like it should be a `TODO` issue.

Reply:
It probably should, I'm inclined to think maybe we should reduce our docs and point to emulator docs somewhere else? We're mostly not recommending this path right now.

Reply:
Yeah - either is fine by me, so long as we log a follow-up issue.

_(It can wait until after Public Preview, if need be.)_